src/checkpoint/fevm_fvm/topdown.rs

```diff
@@ -126,9 +126,17 @@ impl<P: EthManager + Send + Sync, C: LotusClient + Send + Sync> CheckpointManage
         epoch: ChainEpoch,
         validator: &Address,
     ) -> anyhow::Result<()> {
+        let nonce = gateway_state(&self.child_fvm_manager, &self.child)
```

Comment:
Can you remind me what is the purpose of this `gateway_state` function?

Reply:
just get the state of IPC Gateway actor, this method is used over and over, put it in a function for code reuse.